.github/no-response.yml

```diff
@@ -0,0 +1,8 @@
+
+# Number of days of inactivity before an Issue is closed for lack of response
+daysUntilClose: 7
```

Comment:
Maybe make it like a month? A week isn't too long for someone to fix it themselves

Reply:
I'm only closing issues so I don't have to look at them. A month is practically forever :) I can simply leave them open. I might forget about some issues this way, but I can live with that.